src/make_embeddings.jl

```diff
@@ -0,0 +1,165 @@
+include("parser.jl")
```

Comment:
Do not import files in each other. Try to keep it flat and write all the imports in `RAGKit.jl`

Reply:
By import I mean `include()`